### PR TITLE
feat: Configure Sentry crash reporting

### LINKED ISSUE #106: Crash Monitoring with Sentry

### PR DESCRIPTION
**Pull request by sdsantos**
Closes #106

### CODE REVIEW

**Review by aanorbel — APPROVED** [review]
🐳

**Comment by aanorbel** [comment]
Please fix broken test

File: composeApp/build.gradle.kts [hunk 1]
```diff
@@ -88,7 +88,7 @@ kotlin {
     iosSimulatorArm64()
 
     cocoapods {
-        ios.deploymentTarget = "12.0"
+        ios.deploymentTarget = "15.3"
```
Comment (aanorbel) [inline composeApp/build.gradle.kts:91]: If sentry is the only thing forcing us to jump versions, can we use an older version of sentry?

Comment (sdsantos) [reply composeApp/build.gradle.kts:91]: This was me during bug fixing. But I just copied the minimum we had on the iOS config files. I think we had a lower minimum here than there, so I just matched it. Our minimum is 15.3 right?

Comment (aanorbel) [reply composeApp/build.gradle.kts:91]: its `13.0`

Comment (sdsantos) [reply composeApp/build.gradle.kts:91]: Oh, then I have to fix it on a bunch of different places.

Comment (sdsantos) [reply composeApp/build.gradle.kts:91]: Addressed in https://github.com/ooni/probe-multiplatform/pull/133

**Pull request merged**